charts/exivity/templates/edify/service-monitor.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.prometheus.metricServer.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "exivity.fullname" $ -}}-component-edify
```

Comment:
Any suggestion for an alternative name for `component`?
This will be shown within Prometheus metrics as label `service`